README.md

```diff
@@ -142,6 +153,8 @@ or using slashes as path delimiters:
 @assert load("example.jld2", "mygroup/mystuff") == 42
 ```
 
+The group `file["mygroup"]` can be accessed with the same file-like interface as the "full" struct. When using [UnPack.jl](https://github.com/mauro3/UnPack.jl), the macros `@unpack` and `@pack!` work without modifications.
+
```

Comment:
I think I'd prefer to have a separate section in the docs / readme titled something like `UnPack.jl API` 
rather than having it interleaved with the rest.

Personally, I haven't used these macros even once since the regular destructuring syntax was introduced.
(which can't really use here since it uses `getproperty`)

Reply:
Alright, good idea. Your suggestion is implemented in the new commit :+1: